umlfri2/application/drawingarea/selection.py

```diff
@@ -1,5 +1,7 @@
-from collections import Iterable
-
+try:
+    from collections.abc import Iterable
+except ImportError:
```

Comment:
Fallback for the old import is not needed as all supported versions of Python 3 contains abc module.